showcase/src/assets/examples/breadcrumb/breadcrumb-no-input.ts

```diff
@@ -0,0 +1,21 @@
+import { Component, Inject, OnInit } from "@angular/core";
+import { STARK_LOGGING_SERVICE, StarkLoggingService } from "@nationalbankbelgium/stark-core";
+const componentName: string = "StarkBreadcrumb";
+
+/**
+ * Demo Breadcrumb with no input component page
+ */
+@Component({
+	selector: "breadcrumb-component",
```

Comment:
The selector of a component should not contain "component" word.
Could you please change it for "breadcrumb" or change the component name for DemoBreadcrumbComponent and change the selector for "demo-breadcrumb"